CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -1156,9 +938,26 @@ public int CreatePipeline() {
                 VkPipelineDynamicStateCreateInfo dynamicState = VkPipelineDynamicStateCreateInfo.calloc();
```

Comment:
Should this be callocStack?

Reply:
oops, yes changed

Reply:
Can you just do a quick search and see what other instances we have that should be callocStack please?

Reply:
Found two more and changed them